linux/scripts/linux-workshop-admin.sh

```diff
@@ -27,28 +27,52 @@ score-for-step() {
     if [[ -f "${wsroot}/instructions/step_${next_step}.md" ]] ; then
       printf 'Providing instruction to user for Step %s\n' "${next_step}"
       cp "${wsroot}/instructions/step_${next_step}.md" /home/appuser/
+      # Also broadcast message to user when step is complete
+      wall "Congrats on finishing step ${which_step}! Be sure to check your home directory for any new instruction files! (hit any key to dismiss this message)"
```

Comment:
It's not a HUGE issue since the participants will still get the message, but `wall` prints out the *same* message several times when invoked, and I haven't been able to figure out how to stop that yet. It seems to only happen when the root user is the invoker -- but if you `sudo -u appuser wall` then it doesn't broadcast to the logged-in user (since it's the same invoker).

Reply:
I'm not entirely sure what you mean "prints several times"

This is what I see when I test completing the steps:
<img width="1368" alt="image" src="https://github.com/opensourcecorp/workshops/assets/29483705/235b797e-e5c4-4a34-b1f1-b7485951fa89">

(Right is appuser, left is admin). Is that different than what you're seeing?

Reply:
oh WEIRD, maybe it's just Vagrant buffering it funny then. This is what I see locally:

![image](https://github.com/opensourcecorp/workshops/assets/22351501/99802438-9200-4d06-8c26-b69129b19294)

But if you're not seeing it, then maybe it's fine lol.